src/Raven.Studio/typescript/models/database/tasks/importDatabaseModel.ts

```diff
@@ -98,7 +85,38 @@ class importDatabaseModel {
             if (customize) {
                 this.includeDatabaseRecord(true);
             }
-        })
+        });
+
+        this.itemsToWarnAbout = ko.pureComputed(() => {
+            let items = [];
+
+            if (!this.includeDocuments()) {
+                if (this.includeAttachments()) {
+                    items.push(["Attachments"]);
+                }
+                if (this.includeCounters()) {
+                    items.push(["Counters"]);
+                }
+                if (this.includeTimeSeries()) {
+                    items.push(["Time Series"]);
+                }
+                if (this.includeRevisionDocuments()) {
+                    items.push(["Revisions"]);
+                }
+            }
+
+            switch (items.length) {
```

Comment:
or maybe let's move this to generalIUtils? as right now we duplicate code + we can support all lengths.

Reply:
or... can we simply join with comma?

Reply:
done - moved to generalIUtils
but I prefer to keep it grammatically correct w/ '&' before the last one